rpcd/playbooks/roles/filebeat/defaults/main.yml

```diff
@@ -50,6 +50,7 @@ filebeat_logstash_hosts: "{% for host in groups['logstash_all'] %}{{ hostvars[ho
 #
 #                              Date        Time       PID     Level   Python Module   Id
 multiline_openstack_pattern: '^[0-9-]{10} +[0-9:\.]+ +[0-9]+ +[A-Z]+ +[A-Za-z0-9\._]+ \[|Traceback'
+multiline_keystone_error_pattern: '^[0-9-]{10} +[0-9:\.]+ [0-9-]{10} +[0-9]+ +[A-Z]+ +[A-Za-z0-9\._]+ \[|Traceback'
```

Comment:
Looking at [this log event](https://104.239.156.122:8443/app/kibana#/doc/logstash-*/logstash-2016.08.09/log?id=AVZwal3bXLZuH-GTFFB4&_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:now-15m,mode:quick,to:now))) this doesn't seem to work